<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>297</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>392</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>oct梦魇触角重塑工具 by wangdonghao</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="rebuild_crv_pb">
     <property name="text">
      <string>rebuild:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="rebuild_crv_sb">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="foot_le"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="foot_pb">
     <property name="text">
      <string>&lt;&lt; 触角</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="make_planes_pb">
         <property name="minimumSize">
          <size>
           <width>208</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>208</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>生成面片</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="make_planes_sb">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Aharoni</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>subdivisions</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="make_loft_crvs_pb">
         <property name="minimumSize">
          <size>
           <width>208</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>208</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>生成loft环形边</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="rebuild_crv_2_sb">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>40</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>rebuild</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="reshape_mesh_pb">
         <property name="minimumSize">
          <size>
           <width>208</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>208</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>loft</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="cv_offset_sb">
         <property name="minimum">
          <number>-99</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cv_offset_pb">
         <property name="text">
          <string>《修正曲线</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="build_cylinder_pb">
         <property name="minimumSize">
          <size>
           <width>208</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>208</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>创建圆筒</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="build_cylinder_sb">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Aharoni</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>圆筒半径</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
